configuration.rst

```diff
@@ -73,17 +80,6 @@ To reload the configuration without restarting the PostgREST server send a SIGUS
 
   killall -SIGUSR2 postgrest
 
-.. note::
-
-   To refresh the cache in docker:
-
-   .. code:: bash
-
-     docker kill -s SIGUSR2 <container>
-
-     # or in docker-compose
-     docker-compose kill -s SIGUSR2 <service>
```

Comment:
According to these comments https://github.com/PostgREST/postgrest/pull/1691#discussion_r542065652 this isn't possible in Docker now that no config file is used. After #422 is merged, there should be a reference to In-db configuration reloading instead.

Reply:
Yeah, true. This should be removed.